pkgs/os-specific/linux/cpuset/default.nix

```diff
@@ -1,27 +1,42 @@
 { stdenv
 , fetchFromGitHub
-, python2Packages
+, fetchpatch
+, pythonPackages
 }:
 
-python2Packages.buildPythonApplication rec {
+pythonPackages.buildPythonApplication rec {
   pname = "cpuset";
-  version = "1.5.8";
+  version = "1.6";
 
-  propagatedBuildInputs = [ ];
+  propagatedBuildInputs = with pythonPackages; [
+    configparser
+    future
+  ];
+
+  # https://github.com/lpechacek/cpuset/pull/36
+  patches = fetchpatch {
```

Comment:
isn't `patches` usually a list?

Reply:
If I put `fetchpatch` inside a list it returns an error.

Reply:
ah, ok then.

Reply:
Please use this style:

https://github.com/NixOS/nixpkgs/blob/e7fe577d9803885d1191c6612b95c246cb605dde/pkgs/development/tools/phantomjs2/default.nix#L38